learn/wavemaker-release-notes/v11-2-5.md

```diff
@@ -0,0 +1,105 @@
+---
+title: "WaveMaker 11.2.5 - Release date: 20 March 2023"
+id: "v11-2-5"
+sidebar_label: "v11.2.5 - latest"
+---
+This release includes bug fixes for accessiblity in widgets and In-memory sessions.
+
+---
+
+## Bug fixes
+
+- Fixed accessibility issues in data table, list, form, switch, tab, radio button, pop-over, carousel, calendar, date, checkbox, label and Nav bar widgets.
+- Added setAuthorities method in WMAuthentication class to update or set the authorities.
```

Comment:
This should be linking to the doc: 
https://docs.wavemaker.com/learn/how-tos/customizing-post-authentication-handlers#wmauthentication-class

And add more info about the method where appropriate